torch/quantization/_generated/QAddGen.py

```diff
@@ -0,0 +1,33 @@
+
+# This file is generated using `torch.nn.quantization.make_module`
+# and saved as `QAddGen.py`.
+
+import torch
+
+r"""AddGen wraps the torch.add function."""
+class AddGen(torch.nn.Module):
+    def __init__(self, **kwargs):
```

Comment:
why kwags?

Reply:
To be able to reuse it for different types of ops that could be generated -- it's part of the template.